Themes/default/GenericControls.template.php

```diff
@@ -236,7 +236,7 @@ function template_control_verification($verify_id, $display_type = 'all', $reset
 			{
 				$lang = (isset($txt['lang_recaptcha']) ? $txt['lang_recaptcha'] : $txt['lang_dictionary']);
 				echo '
-				<div class="g-recaptcha centertext" data-sitekey="' . $verify_context['recaptcha_site_key'] . '" data-theme="' . $verify_context['recaptcha_theme'] . '"></div>
+				<div class="google_recaptcha centertext" data-sitekey="' . $verify_context['recaptcha_site_key'] . '" data-theme="' . $verify_context['recaptcha_theme'] . '"></div>
```

Comment:
Please revert this one. "g-recaptcha" comes from Google's own scripts. The script probably doesn't care about the class name of this div, but I'd rather err to the side of safety.